README.md

```diff
@@ -0,0 +1,23 @@
+# CCAO Data Infrastructure
+
+This repository stores the code for the CCAO Data Department's ETL
+pipelines and data lakehouse. This infrastructure supports the Data Team's
+modeling, reporting, and data integrity work.
+
+## Quick Links
+
+- [:file_folder: dbt Data Catalog](https://ccao-data.github.io/data-architecture/#!/overview) -
+  Documentation for all CCAO data lakehouse tables and views
+- [:nut_and_bolt: dbt README](/dbt/README.md) - How to develop CCAO data
+  infrastructure using dbt
+- [:ballot_box_with_check: dbt Test Documentation](/dbt/tests/generic/README.md) -
+  Definitions for CCAO generic dbt data tests
+
+## Repository Structure
+
+- [./dbt](./dbt) contains the models and tests that build our Athena data lakehouse
+- [./docs](./docs) contains design documents and other supplemental documentation
+- [./etl](./etl) contains small ETL scripts used to load data in the lakehouse as
```

Comment:
**[Suggestion, non-blocking]** I feel unsure about the name `etl` for this subdirectory, since our transformations are (mostly) handled elsewhere by dbt. What do you think about `extract-load`?

Reply:
Let's stick with `etl`, but I'll clarify the purpose of each dir in the README.